src/api/app/controllers/webui2/project_controller.rb

```diff
@@ -4,13 +4,9 @@ def webui2_index
     atype = AttribType.find_by_namespace_and_name!('OBS', 'VeryImportantProject')
     @important_projects = Project.find_by_attribute_type(atype).where('name <> ?', 'deleted').pluck(:name, :title)
 
-    if @spider_bot
-      render :list_simple
```

Comment:
Are you also going to remove this action and views etc?

Reply:
everything, yes